### DISCUSSION TITLE
Other themes

### DISCUSSION THREAD

**Discussion by FallenAnother** [Daksh777/SpotifyNoPremium#52] [Q&A] [answered]
Is there a way to use SpotifyNoPremium with other Spicetify themes?

**Comment by Daksh777** [comment, marked as answer]
If the theme is open-source on GitHub, I can send a PR to add this theme with their own theme so you need not do manual work again, you can send the GitHub link here.

And yes, you can copy the lines in `user.css` to the one in any other theme and copy the adblock.js to the extensions folder and run `spicetify config extensions adblock.js` in a terminal

**Reply by FallenAnother** [reply]
Its just one of the themes of the original spicetify pack. Also, is there a way to describe better that process? Im kinda new on that thing and just wanted to have the adblock and a theme on the same time- 

The desired theme is Dribbblish, btw

**Reply by FallenAnother** [reply]
Nevermind, managed to do it! Very much thanks! <3

**Reply by Daksh777** [reply]
What I meant is that I can combine this theme with any other theme, if it's on GitHub. So you can just send me the GitHub link of that theme and I'll do the rest.
